drivers/pulse_counter/pulse_counter.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2017 UC Berkeley
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/**
+ * @ingroup     drivers_pulse_counter
+ * @{
+ *
+ * @file
+ * @brief       Driver for the PULSE COUNTER.
+ *
+ * @author      Hyung-Sin Kim <hs.kim@cs.berkeley.edu>
+ *
+ * @}
+ */
+
+#include <string.h>
+
+#include "pulse_counter_params.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+/**
+ * @brief   Define the number of configured sensors
+ */
+#define PULSE_COUNTER_NUM    (sizeof(pulse_counter_params)/sizeof(pulse_counter_params[0]))
+
+uint16_t pulse_count[PULSE_COUNTER_NUM];
+
+/* Accumulate pulse count */
+void pulse_counter_trigger(void* arg) {
```

Comment:
This function should be defined `static`.

Reply:
fixed. Thanks!